### ISSUE TITLE
Say more than just "dhcpcd is not running"

### ISSUE THREAD

**Issue by jidanni** [NetworkConfiguration/dhcpcd#274]
```
# dhcpcd --release
dhcpcd is not running
# pidof dhcpcd
3220 3219 3218 3217
```

```
# dhcpcd --exit
dhcpcd is not running
# pidof dhcpcd
3556 3532 3514 3511 3220 3219 3218 3217
```

Say perhaps ```dhcpcd is not running on the interface ("") you provided.``

**Comment by rsmarples** [comment]
It's also split by address family.
You could do `dhcpcd -n -4 eth0` and then `dhcpcd --release eth0` would give the same error.

This was actually fixed in e337bd7ce4a8 and released in dhcpcd-10.0.6 recently.

Also note for dhcpcd-11 this will all change as you won't be able to start dhcpcd like that anymore.
See #271.

**Comment by jidanni** [comment]
> won't be able to start dhcpcd like that anymore

I didn't start it. All I did was check it.

I currently start it with
```
dhcpcd wlx88888888
```
Will that still work in the future? Thanks.

**Comment by rsmarples** [comment]
Yes it will. The important thing is that the error you describe above will not happen anymore.

**Comment by perkelix** [comment]
Does the above still apply to 10.5.2?

**Comment by jidanni** [comment]
I don't know.
This is now complex for me.
And I don't want to shut down my connection to test.
